scripts/postcss/css-url-processor.js

```diff
@@ -39,7 +39,7 @@ function colorsFromURL(url, colorMap) {
 function processURL(decl, replacer, colorMap) {
     const value = decl.value;
     const parsed = valueParser(value);
-    parsed.walk(async node => {
+    parsed.walk(node => {
```

Comment:
This was previously marked as `async` but `await` was never used here so it wasn't necessary. And had the side-effect of swallowing any error that occurred within the plugin so it looked like the asset build succeeded even though it didn't.